integration-test/src/test/java/org/apache/iotdb/db/it/IoTDBConfigNodeIT.java

```diff
@@ -204,6 +228,597 @@ public void removeAndStopConfigNodeTest() {
+      assertEquals(4, infoMap.size());
+      List<Map.Entry<Integer, TDataNodeConfiguration>> infoList =
+          new ArrayList<>(infoMap.entrySet());
+      infoList.sort(Comparator.comparingInt(Map.Entry::getKey));
+      int count = 0;
+      for (DataNodeWrapper expectedDataNode : dataNodeWrappers) {
+        for (int i = 0; i < 4; i++) {
+          TDataNodeLocation dnLocation = infoList.get(i).getValue().getLocation();
+          if (expectedDataNode.getInternalPort() == dnLocation.getInternalEndPoint().getPort()
+              && expectedDataNode.getMppDataExchangePort()
+                  == dnLocation.getMPPDataExchangeEndPoint().getPort()
+              && expectedDataNode.getSchemaRegionConsensusPort()
+                  == dnLocation.getSchemaRegionConsensusEndPoint().getPort()
+              && expectedDataNode.getDataRegionConsensusPort()
+                  == dnLocation.getDataRegionConsensusEndPoint().getPort()) {
+            count++;
+            break;
+          }
+        }
+      }
+      assertEquals(4, count);
+
+      // test query one DataNodeInfo
+      dataNodeConfigurationResp = client.getDataNodeConfiguration(dataNodeLocation.getDataNodeId());
+      infoMap = dataNodeConfigurationResp.getDataNodeConfigurationMap();
+      TDataNodeLocation dnLocation =
+          dataNodeConfigurationResp
+              .getDataNodeConfigurationMap()
+              .get(dataNodeLocation.getDataNodeId())
+              .getLocation();
+      assertEquals(
+          TSStatusCode.SUCCESS_STATUS.getStatusCode(),
+          dataNodeConfigurationResp.getStatus().getCode());
+      assertEquals(1, infoMap.size());
+      assertEquals(dataNodeLocation, dnLocation);
+
+      // test remove DataNode
+      removeDataNodeLocationList.add(dataNodeLocation);
+      removeReq = new TDataNodeRemoveReq(removeDataNodeLocationList);
+      removeResp = client.removeDataNode(removeReq);
+      assertEquals(TSStatusCode.SUCCESS_STATUS.getStatusCode(), removeResp.getStatus().getCode());
+
+      for (int i = 0; i < retryNum; i++) {
+        showDataNodesResp = client.showDataNodes();
+        if (showDataNodesResp.getDataNodesInfoListSize() == 3) {
+          break;
+        }
+        Thread.sleep(1000);
+      }
+      assertEquals(3, showDataNodesResp.getDataNodesInfoListSize());
+
+      dataNodeInfos = showDataNodesResp.getDataNodesInfoList();
+      for (TDataNodeInfo dnInfo : dataNodeInfos) {
+        assertNotEquals(dataNodeLocation.getDataNodeId(), dnInfo.getDataNodeId());
+      }
+      dataNodeConfigurationResp = client.getDataNodeConfiguration(dataNodeLocation.getDataNodeId());
+      assertEquals(
+          TSStatusCode.SUCCESS_STATUS.getStatusCode(),
+          dataNodeConfigurationResp.getStatus().getCode());
+      assertEquals(0, dataNodeConfigurationResp.getDataNodeConfigurationMap().size());
+    } catch (Exception e) {
+      e.printStackTrace();
+      fail(e.getMessage());
+    }
```

Comment:
Just throw any Exceptions in the interface, we don't need to catch and print them since it's a test code